src/Storages/System/StorageSystemDashboards.cpp

```diff
@@ -349,6 +363,11 @@ ORDER BY t WITH FILL STEP {rounding:UInt32}
             { "dashboard", "Cloud overview" },
             { "title", "Network send bytes/sec" },
             { "query", "SELECT toStartOfInterval(event_time, INTERVAL {rounding:UInt32} SECOND)::INT AS t, avg(value)\nFROM (\n  SELECT event_time, sum(value) AS value\n  FROM clusterAllReplicas(default, merge('system', '^asynchronous_metric_log'))\n  WHERE event_date >= toDate(now() - {seconds:UInt32})\n    AND event_time >= now() - {seconds:UInt32}\n    AND metric LIKE 'NetworkSendBytes%'\n  GROUP BY event_time)\nGROUP BY t\nORDER BY t WITH FILL STEP {rounding:UInt32} SETTINGS skip_unavailable_shards = 1" }
+        },
+        {
+            { "dashboard", "Cloud overview" },
+            { "title", "Maximum concurrent network connections" },
```

Comment:
The maximum is taken on every rounding interval, e.g., on a ten-second interval, for downsampling. The graph will display the dynamics over time - it is not correct to name the whole chart as a "maximum".

Just say "Concurrent network connections"

Reply:
ok thanks for the feedback and suggestion, will update